### ISSUE TITLE
escaped! missing documentation

### ISSUE THREAD

**Issue by Aaronepower** [Geal/nom#131]
http://rust.unhandledexpression.com/nom/macro.escaped!.html

**Comment by Geal** [comment]
Fixed in e6cf389

**Comment by Kerollmops** [comment]
It seems escaped_transform has been updated and need one more argument but I can't figure out what.
```
error: expected ident, found '\\'
  --> src/lib.rs:12:34
   |
12 | escaped_transform!(call!(alpha), '\\',
   |                                  ^^^^
```
